addons/point_of_sale/static/src/js/ChromeWidgets/SyncNotification.js

```diff
@@ -13,6 +14,10 @@ export class SyncNotification extends Component {
         return this.pos.globalState.synch;
     }
     onClick() {
+        if (this.pos.globalState.synch.status !== "connected") {
+            this.pos.globalState.showOfflineWarning = true;
+            throw new ConnectionLostError();
+        }
```

Comment:
I find it strange as you are not making any request. The PoS cannot know if it is connected or not, leading to a connection error popup even if the user is reconnected to the network.

Reply:
You might want to do a try catch on the orm call to see if the connection is back or not and act accordingly to this information.